pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,19 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// CephBlockPoolRadosNamespaceSpecMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type CephBlockPoolRadosNamespaceSpecMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemotePool is the name of the secondary cluster pool
```

Comment:
```suggestion
	// RemoteNamespace is the name of the radosNamespace on the secondary cluster pool
```
Or something similar?